src/coreComponents/physicsSolvers/wavePropagation/WaveSolverBase.cpp

```diff
@@ -24,6 +24,7 @@
 #include "fieldSpecification/FieldSpecificationManager.hpp"
 #include "mainInterface/ProblemManager.hpp"
 #include "mesh/mpiCommunications/CommunicationTools.hpp"
+#include "mainInterface/GeosxState.hpp"
```

Comment:
You may not have to include this file here, but I am not sure. Do you remember why you included it?

Reply:
Maybe this one was to access ProblemManager through GeosxSate, not sure. Since we don't use it anymore I'll remove it.
Thanks